evaluation/swe_bench/run_infer.py

```diff
@@ -88,6 +89,13 @@ def get_instruction(instance: pd.Series, metadata: EvalMetadata):
             '5. Think about edgecases and make sure your fix handles them as well\n'
             "Your thinking should be thorough and so it's fine if it's very long.\n"
         )
+
+    if RUN_WITH_BROWSING:
+        instruction += (
+            '<IMPORTANT!>\n'
+            'You SHOULD NEVER attempt to browse the web. '
+            '</IMPORTANT!>\n'
+        )
```

Comment:
Is there a way we can keep track of statistics of how often this happened? We're not guaranteed that the agent will follow these directions, so at least being able to post-hoc audit whether it happened or not would be good.

Reply:
Yeah! It is not hard - and simple cat with jq should be sufficient

`cat output.jsonl | jq '.history[].action' | sort | uniq -c`

<img width="1813" alt="image" src="https://github.com/user-attachments/assets/4333bcc2-a787-4a78-961d-d925b96de54c">
Did a quick check - there's no `browse_interactive` action in our SWE-Bench run